cpp/doxygen/developer_guide/DEVELOPER_GUIDE.md

```diff
@@ -1483,6 +1483,16 @@ struct, and therefore `cudf::struct_view` is the data type of a `cudf::column` o
 
 `cudf::type_dispatcher` dispatches to the `struct_view` data type when invoked on a `STRUCT` column.
 
+# Empty Columns
+
+The libcudf columns support empty, typed content. These columns have no data and no validity mask.
+Empty strings columns may or may not contain a child offsets column. If is undefined behavior (UB)
+to access the children of an empty strings column. Nested columns like lists and structs may
```

Comment:
is there any behavior like this in lists columns?

Reply:
Do you mean accessing children for an empty lists column?
I think the next sentence describes lists and structs may have children.

Reply:
I wondering whether a lists column created with `empty_like()` always has an offset child column.

Reply:
It looks like it does but it would probably be prudent to say accessing it UB as well.